admins.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import urllib
+
+import requests
+
+from client import get_token, get_github3_client
+
+if __name__ == '__main__':
+    # only using this for members_urlt
+    gh = get_github3_client()
```

Comment:
It does. I'm 50/50 on it - would be nice not to repeat the members URL template in the library and in the app code, but ... :neutral_face: